referenceframe/frame.go

```diff
@@ -400,13 +439,18 @@ func (rf *rotationalFrame) Name() string {
 }
 
 func (rf *rotationalFrame) MarshalJSON() ([]byte, error) {
-	m := FrameMapConfig{
-		"type":    "rotational",
-		"name":    rf.name,
-		"rotAxis": rf.rotAxis,
-		"limit":   rf.limits,
+	if len(rf.limits) > 1 {
+		return nil, errors.New("cannot marshal revolute frame with >1 DOF, use a Model")
```

Comment:
if we use this error message a couple places lets just make it a constant or function